docs/meetings/community/2023-01-17.md

```diff
@@ -0,0 +1,167 @@
+was here. Roll call:
+
+- Min / Simula / @minrk (will be late)
+- Blessing / Outreachy / @bl-aire
+- Sarah / 2i2c / @sgibson91
+- Erik / 2i2c / @consideRatio
+- Simon / University of Dundee / @manics
+- Eskild / Quansight / @iameskild
+- Wayne / Upstate Medical University / @fomightez
+- Yuvi / 2i2c / @yuvipanda
+- Sheila / Outreachy / @Sheila-nk
+
+## Introductions
+
+If this is your first time attending the meeting, add your name below and
+we will ask you to introduce yourself on the call :)
+
+- Morgan Radic
+- Sheila Kahwai
+
+## Quick updates
+
+60 second updates on things you have been up to, questions you have, or
+developments you think people should know about. Please add yourself, and
+if you do not have an update to share, you can pass.
+
+- **Min:** move away from problematic referer checks to xsrf tokens
+  ready for review: https://github.com/jupyterhub/jupyterhub/pull/4032
+  (thanks for Oksana for selenium tests that help ensure actual pages work,
+  not just test requests)
+
+## Reports and celebrations
+
+This is a place to make announcements (without a need for discussion).
+This is also a great place to give shout-outs to contributors! We'll read
+through these at the beginning of the meeting.
+
+- **Erik:** `@dolfinus` has made a lot of thorough PRs to KubeSpawner - 
+such
+  as adding tests! ([List of
+  PRs](https://github.com/jupyterhub/kubespawner/pulls?q=is%3Apr+author%3Adolfinus+))
+- **Erik:** `@bollwyvl` for work in jupyterhub/jupyter-server-proxy 
+regarding
+  tests!
+
+## Agenda items
+
+Let's collect all potential agenda items here before the start of the
+meeting. We will then attempt to create a coherent agenda that fits in the
+60m meeting slot. If there are similar items try and group them together.
+
+- **Min** (5m): Outreachy for Spring. Do we do it, or wait for Fall? We
+  need project ideas, mentors and especially volunteers for the application
+  period. First deadline is already Feb 10, which is before our next
+  meeting! <https://github.com/jupyterhub/team-compass/issues/612>
+  - Money from Outreachy for two interns
+  - We need mentors to help detail projects for the interns
+- **Sarah** (probably following on from Min's topic): What does the
+  JupyterHub community want the Strategic Lead to focus on? What outside
+  Outreachy is important to prioritise? Is Outreachy a top priority or not?
+  - Sarah acting the community coordinator
+  - Sarah concerned about the additional work that Outreachy would
+    involve, would there be other community things that you would prefer 
+Sarah work on?
+  - Sarah has been working on [Outreachy
+    guide](https://jupyterhub-outreachy.readthedocs.io/en/latest/).
+  - **Simon:** what are other things Sarah could spend time on?
+    - Restructure this meeting. Can be challenging for new attendees.
+      Proposal, two meetings
+      - Collaboration cafe:
+        - Sarah's hackmd: <https://hackmd.io/@sgibson91/SyD50E2qo>
```

Comment:
This link is private

Reply:
It was set to "Signed-in users", changed now